BackEnd/tests/DeleteProjectTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace HostMyDocs\Tests;
+
+class DeleteProjectTest extends BaseTestCase
+{
+    /**
+     * @var array credentials to be inserted into environment
+     */
+    private $serverCredentials = 'beep:bep';
+
+    private $wrongCredentials = 'plop:plop';
+
+    private static $isInitialized = false;
+
+    public function setUp()
+    {
+        // allow setUp to be done once
+        if (self::$isInitialized) {
```

Comment:
Why using this variable ?

Reply:
this allow to make the `setUp()` only once because PHPUnit has a `setUpBeforeClass()` but it is static only, this solution come from [this SO comment](https://stackoverflow.com/questions/7158930/phpunit-how-do-i-create-a-function-to-be-called-once-for-all-my-tests#comment70152044_7159033)